core/src/main/java/org/jruby/ext/socket/SocketUtils.java

```diff
@@ -130,58 +130,6 @@ public static IRubyObject getservbyname(ThreadContext context, IRubyObject[] arg
         return runtime.newFixnum(port);
     }
 
-    public static IRubyObject pack_sockaddr_in(ThreadContext context, IRubyObject port, IRubyObject host) {
```

Comment:
@etehtsea Since these are public static methods we should deprecate and call to the new location since there is a small possibility some Java Native Extension could be calling them.